example/eudi_wallet_python_backend.yml

```diff
@@ -0,0 +1,137 @@
+module: backends.openidvp4saml2.OpenIDVP4SAMLBackend
```

Comment:
SAML?

Why SAML?

please remove
I have suggested something like

`eudi_wallet_python.satosa.backends.openid4vp`

Reply:
Fixed in f3b239c2f125e191f25635abbccac65c306fafc0 and 74408287a6afb32e011daeb44dcaf0fe614b64d1